### PR TITLE
Add RE2, RB20, 22, 24 ,25, RE1, RE6, RB 45, RB 51, RB53

### PR DESCRIPTION
**Pull request by NxrVincent**
Added RE2 (DB Regio BaWü)
Added RB20,22,24,25 (SWEG)
Added RE1, RE6, RB 45, RB 51, RB53  (DB Regio Mitte)

Updated SWEG S5, S3 & S2 to the right colors

### CODE REVIEW

**Comment by NxrVincent** [comment]
Oh, sorry for the JSON error. Just added it quickly via GitHub Web. Thanks :)!

**Review by marhei — CHANGES_REQUESTED** [review]
Can you please add yourself as contributor of db-regio-mitte, rvf-sbahn, sweg-sudwest and tlx in the sources.json file.

File: line-colors.csv [hunk 1]
```diff
@@ -1129,7 +1137,11 @@ sweg-sudwest,RB 66,sweg-sudwestdeutsche-landesverkehrs-gmbh,swe-rb66,#014f9f,#ff
 sweg-sudwest,RB 67,sweg-sudwestdeutsche-landesverkehrs-gmbh,swe-rb67,#00a9df,#ffffff,,rectangle
 sweg-sudwest,RB 68,sweg-sudwestdeutsche-landesverkehrs-gmbh,swe-rb68,#ec2933,#ffffff,,rectangle
 sweg-sudwest,RB 69,sweg-sudwestdeutsche-landesverkehrs-gmbh,swe-rb69,#f27320,#ffffff,,rectangle
-tlx,L2,trilex-die-landerbahn-gmbh-dlb,tl-l2,#0f7ec0,#ffffff,,rectangle-rounded-corner
+sweg-sudwest,RB25,sweg-sudwestdeutsche-landesverkehrs-gmbh,3-s6-rb25,#b6931d,#ffffff,,rectangle
+sweg-sudwest,RB20,sweg-sudwestdeutsche-landesverkehrs-gmbh,3-s6-rb20,#d30630,#ffffff,,rectangle
+sweg-sudwest,RB24,sweg-sudwestdeutsche-landesverkehrs-gmbh,3-s6-rb24,#afca0b,#ffffff,,rectangle
+sweg-sudwest,RB22,sweg-sudwestdeutsche-landesverkehrs-gmbh,3-s6-rb22,#b15a9e,#ffffff,,rectangle
+tlx,RB 3-s6-rb20,trilex-die-landerbahn-gmbh-dlb,tl-l2,#0f7ec0,#ffffff,,rectangle-rounded-corner
```
Comment (marhei) [inline line-colors.csv:1144]: Is this right?

Comment (NxrVincent) [reply line-colors.csv:1144]:
```
sweg-sudwest,RB20,sweg-sudwestdeutsche-landesverkehrs-gmbh,3-s6-rb20,#d30630,#ffffff,,rectangle
sweg-sudwest,RB24,sweg-sudwestdeutsche-landesverkehrs-gmbh,3-s6-rb24,#afca0b,#ffffff,,rectangle
sweg-sudwest,RB22,sweg-sudwestdeutsche-landesverkehrs-gmbh,3-s6-rb22,#b15a9e,#ffffff,,rectangle
```

is correct, but I have not changed anything in tlx

**Review by marhei — APPROVED** [review]

**Pull request merged**